src/content/docs/en/getting-started.mdx

```diff
@@ -7,9 +7,9 @@ import Button from '~/components/Button.astro'
 import ContributorList from '~/components/ContributorList.astro'
 import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
 
-:::tip[Astro version 2 is here!]
-**What’s new in Astro v2?** [Find out in the announcement blog post](https://astro.build/blog/astro-2/).  
-**Ready to upgrade?** [Follow our upgrade guide](/en/guides/upgrade-to/v2/).
+:::tip[Astro version 2.2 has arrived!]
+**New in Astro v2.2:** [CDN support &  `image()` schema API change](https://astro.build/blog/astro-220/).  
+Still on Astro v1? [Read our v2.0 announcement post](https://astro.build/blog/astro-2/) and [follow our upgrade guide](/en/guides/upgrade-to/v2/).
 :::
```

Comment:
```suggestion
```
Hope it's OK, I'm just going to remove the note entirely and commit. Thanks so much for getting the ball rolling on a change we definitely wanted to make!  :raised_hands:  Welcome to Team Docs!  :partying_face: